mangaki/mangaki/settings.py

```diff
@@ -7,10 +7,13 @@
 https://docs.djangoproject.com/en/1.9/ref/settings/
 """
 
-import os
 import configparser
 import json
+import os
+
 BASE_DIR = os.path.dirname(os.path.dirname(__file__))
+FIXTURE_DIR = os.path.join(os.path.dirname(BASE_DIR), 'fixtures')
+TEST_FIXTURE_DIR = os.path.join(FIXTURE_DIR, 'tests')
```

Comment:
I don't think these should be in `fixtures`; this has a well-defined meaning in Django (initial data for models) which those definitely are not. I'd put it in a `data/` subfolder in the `tests/` directory, since those are meant to be used by tests only.